src/responder/secrets/local.c

```diff
@@ -450,6 +451,25 @@ static int local_db_check_number_of_secrets(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static int local_check_max_payload_size(struct local_context *lctx,
+                                        int payload_size)
+{
+    int max_payload_size;
+
+    max_payload_size = lctx->max_payload_size * 1024; /* kb */
+    if (payload_size > max_payload_size) {
+        DEBUG(SSSDBG_OP_FAILURE,
+              "Secrets' payload size [%d kb (%d)] exceeds the maximum allowed "
+              "payload size [%d kb (%d)]\n",
+              payload_size * 1024 /* kb */, payload_size,
```

Comment:
could you split the debug message so it has one parameter per line? For some reason my eyes always stop reading when I notice the comment :)